examples/feature_selection/plot_feature_selection_pipeline.py

```diff
@@ -1,40 +1,92 @@
 """
 ==================
-Pipeline Anova SVM
+Pipeline ANOVA SVM
 ==================
 
-Simple usage of Pipeline that runs successively a univariate
-feature selection with anova and then a SVM of the selected features.
+This example shows how a feature selection can be easily integrated within
+a machine learning pipeline.
 
-Using a sub-pipeline, the fitted coefficients can be mapped back into
-the original feature space.
+We also show that you can easily introspect part of the pipeline.
 """
-from sklearn import svm
+# %%
+print(__doc__)
+from sklearn import set_config
+set_config(display='diagram')
+
+# %%
+# We will start by generating a binary classification dataset. Subsequently, we
+# will divide the dataset into two subsets.
+
+# %%
 from sklearn.datasets import make_classification
-from sklearn.feature_selection import SelectKBest, f_classif
-from sklearn.pipeline import make_pipeline
 from sklearn.model_selection import train_test_split
-from sklearn.metrics import classification_report
 
-print(__doc__)
-
-# import some data to play with
 X, y = make_classification(
```

Comment:
We need to set random state here otherwise the feature selected is not deterministic.